src/validation.h

```diff
@@ -150,6 +151,8 @@ extern bool fPruneMode;
 /** Number of MiB of block files that we're trying to stay below. */
 extern uint64_t nPruneTarget;
 
+extern const std::vector<std::string> CHECKLEVEL_DOC;
```

Comment:
A doxygen comment at line 153 might be nice.

Reply:
Doxygen comment added